act-rules-format.bs

```diff
@@ -593,38 +593,51 @@ Appendix 1: Expressing ACT Rule results with JSON-LD and EARL {#appendix-data-ex
 
 This section is *non-normative*.
 
-**Example 1:** Output data using EARL and JSON-LD. (See [Evaluation and Report Language](https://www.w3.org/WAI/standards-guidelines/earl/) [[EARL10-Schema]] and [Java Script Object Notation (JSON)](https://www.json.org).)
+This section provides examples of expressing results from carrying out ACT Rules using EARL and JSON-LD (See [Evaluation and Report Language](https://www.w3.org/WAI/standards-guidelines/earl/) and [A JSON-based Serialization for Linked Data (JSON-LD)](https://www.w3.org/TR/json-ld/)). More examples and background is proivided on the [JSON-LD serialization of EARL](https://github.com/w3c/earl) GitHub repository.
+
+Examples in this section include:
+
+- Example 1: Minimal outcome from one 'Assertion'
+- Example 2: Results from more than one 'Assertion'
+- Example 3: Aggregating based on 'Requirement'
+- Example 4: Aggregating based on 'Test Subject'
+- Example 5: Assumed 'Context' for this section
+
+**Example 1:** Minimal outcome from one 'Assertion'
 
 ```javascript
 {
-  "@context": "https://raw.githubusercontent.com/w3c/wcag-act/master/earl-act.json",
+  "@context": "context.json",
   "@type": "Assertion",
+  "assertor": "MyTool",
   "subject": "https://example.org/",
-  "test": "auto-wcag:rules/SC1-1-1-css-image.html",
+  "test": "ACT-CG:rules/23a2a8",
   "result": {
     "outcome": "earl:failed",
     "pointer": "html > body > h1:first-child"
```

Comment:
You need to replace the `pointer` property in the context (my preference) with:

```json
"pointer": {
  "@id": "earl:pointer",
  "@type": "ptr:CSSSelectorPointer",
}
```

Or you need to change this to:

```json
"pointer": {
  "@type": "ptr:CSSSelectorPointer",
  "html > body > h1:first-child"
}
```

Reply:
I chose the second approach because the type of pointer should not be controlled by the context but by each assertion separately.